classes/system/heka/remote_collector/single.yml

```diff
@@ -1,6 +0,0 @@
-classes:
-- service.heka.remote_collector.single
-parameters:
-  collectd:
-    client:
-      remote_collector: true
```

Comment:
this needs to be retained otherwise the remote collectd plugins aren't going to be configured.